src/nbl/video/CVulkanCommandBuffer.cpp

```diff
@@ -8,6 +8,24 @@
 using namespace nbl;
 using namespace nbl::video;
 
+namespace
+{
+    VkPipelineBindPoint vkCast(asset::E_PIPELINE_BIND_POINT bindPoint)
```

Comment:
1. why the private namespace
2. I don't remember calling the all my other functions `vkCast` I'm sure I included some of the Vk enum in the name